README.md

```diff
@@ -133,6 +136,10 @@ Give a ⭐️ if you like this project!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🙏 Acknowledgments
+
+> Give credit to everyone who inspired your codebase
+
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>

```

Comment:
- [ ] Kindly provide a license link for your project.

Reply:
@andersonlebon  
Thanks for your review and feedback I will add the marked parts